parlai/mturk/core/mturk_manager.py

```diff
@@ -573,6 +573,7 @@ def _on_alive(self, pkt):
             # Reconnecting worker
             agent = self.worker_manager._get_agent(worker_id, assign_id)
             agent.log_reconnect()
+            conversation_id = agent.conversation_id  # Hot fix suggested by Jack 11/20
```

Comment:
@JackUrb still needed?

Reply:
This has been implemented in full back in #1321 and updated for correctness since